resources/puppetlabs/lein-ezbake/template/global/ext/cli/stop.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set +e
+
+pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
```

Comment:
Hmm, I noticed that for CentOS 7, the full package name is `procps-ng` whereas it is only `procps` for CentOS 6.  For a package I built for CentOS 7 where the dependency was expressed as `procps`, it did appear that the dependency was properly satisfied, though.  Would it be best to run with the shorter name to keep it the same for all RedHat derivatives or should we somehow be more specific per OS?